tests/e2e/tests/07-scale-down-with-filter/02-update-crd.yaml

```
@@ -2,12 +2,18 @@
+  # And also to check if promethius was up for the cooldown period
+  # A low value will make metrics not available for cooldown period
+  cooldownPeriod: 30
+  minTargetReplicas: 1
   triggers:
     - metadata:
         query: round(sum(rate(some_metric[20s])),0.001) or vector(0)
         serverAddress: http://prometheus-operated.monitoring.svc.cluster.local:9090
         threshold: "0.01"
         uptimeFilter: container="foo"
       type: prometheus
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)